openfe/tests/protocols/openmm_septop/test_septop_protocol.py

```diff
@@ -696,24 +696,6 @@ def test_dry_run_benzene_toluene(benzene_toluene_dag, tmpdir):
         assert pdb.n_atoms == 2713
 
 
-def test_output_full_system(
```

Comment:
You would still want a test for the new settings object that checks you can't assign 'all'